app/controllers/deidentified_clients_controller.rb

```diff
@@ -0,0 +1,66 @@
+class DeidentifiedClientsController < ApplicationController
+  before_action :require_can_enter_deidentified_clients! 
+  before_action :require_can_manage_deidentified_clients!, only: [:edit, :update, :destroy]
+  before_action :load_deidentified_client, only: [:edit, :update, :destroy]
+  
+  def index
+    @deidentified_clients = deidentified_client_source.page(params[:page]).per(25)
+  end
+
+  def create
+    clean_params = clean_params(deidentified_client_params)
+    @deidentified_client = deidentified_client_source.create clean_params
+    respond_with(@deidentified_client, location: deidentified_clients_path)
+  end
+  
+  def new
+    @deidentified_client = deidentified_client_source.new 
+  end
+  
+  def edit
+  end
+
+  def update
+    clean_params = clean_params(deidentified_client_params)
```

Comment:
This needs to check the can change cohorts permission and remove the cohorts parameter if you can't.